storybook/src/components/TextInput/TextInput.stories.tsx

```diff
@@ -33,20 +32,51 @@ type Story = StoryObj<typeof meta>
 
 export const Default: Story = {}
 
+export const Password: Story = {
+  args: {
+    minLength: 8,
+    type: 'password',
+    value: 'password',
+  },
+}
+
+export const EmailAddress: Story = {
+  args: {
+    type: 'email',
+    value: 'designsystem@amsterdam.nl',
```

Comment:
Potential spam magnet?

Reply:
I could use user@example.com, but we mention our e-mail address on various other places already.